ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -210,12 +210,7 @@ open class BadgeView: UIView, TokenizedControlInternal {
     }
 
     private func updateFonts() {
-        switch sizeCategory {
-        case .small:
-            label.font = tokenSet.fluentTheme.typography(.caption1)
-        case .medium:
-            label.font = tokenSet.fluentTheme.typography(.body2)
-        }
+        label.font = tokenSet[.labelFont].uiFont
```

Comment:
Since `label` is a Fluent `Label`, could you change this to `label.tokenSet.setOverrideValue(tokenSet.overrideValue(forToken: .labelFont), forToken: .font)` and set the base from by changing the `label`'s style?

Reply:
I think the way we set colors is fine as is, though. `BadgeView` has a lot more states than `Label`, so just setting colors from the `BadgeView`'s tokens makes sense. But we should probably be able to pick up any general font changes from `Label`'s tokens.